.github/workflows/reporting-server.yml

```diff
@@ -0,0 +1,41 @@
+name: reporting-server
+on:
+  pull_request:
+    paths:
+      - 'packages/reporting-server/**'
```

Comment:
Is there any integration test between the api server currently? If so we should add the api server to the list of paths.

Reply:
Actually, there is not. The `api-server` doesn't know the existence of the _reporting-server_. It all goes though fluentd